writing_tests.md

```diff
@@ -0,0 +1,16 @@
+_(in fact, this would be part of the new "docs" repo, not a file in this repo.)_
```

Comment:
Part of this was stolen by "someone" and placed in the TRACK_README. I agree a part of it should also go into docs but some of this about randomization is relevant here.